db/db.js

```diff
@@ -375,33 +374,31 @@ async function isOverdoseSuspected(xethru, session, location, clientParam) {
     const sessionDuration = (now - start_time_sesh) / 1000
 
     // threshold values for the various overdose conditions
-    const rpm_threshold = location.rpmThreshold
     const still_threshold = location.stillThreshold
     const sessionDuration_threshold = location.durationThreshold
 
     // number in front represents the weighting
     // eslint-disable-next-line eqeqeq
-    const condition1 = 1 * (xethru.rpm <= rpm_threshold && xethru.rpm != 0)
-    const condition2 = 1 * (session.stillCounter > still_threshold) // seconds
-    const condition3 = 1 * (sessionDuration > sessionDuration_threshold)
+    const condition1 = 1 * (session.stillCounter > still_threshold) // seconds
+    const condition2 = 1 * (sessionDuration > sessionDuration_threshold)
 
     let alertReason = ALERT_REASON.NOTSET
 
     // eslint-disable-next-line eqeqeq
-    if (condition3 === 1) {
+    if (condition2 === 1) {
       alertReason = ALERT_REASON.DURATION
     }
 
     // eslint-disable-next-line eqeqeq
-    if (condition2 === 1 || condition1 == 1) {
+    if (condition1 == 1) {
       alertReason = ALERT_REASON.STILLNESS
     }
 
     const conditionThreshold = 1
 
     // This method just looks for a majority of conditions to be met
     // This method can apply different weights for different criteria
-    if (condition1 + condition2 + condition3 >= conditionThreshold) {
+    if (condition1 + condition2 >= conditionThreshold) {
```

Comment:
We never ended up using this weighting. 
What do you think about getting rid of this concept and using more readable names for the conditions and ditching `conditionThreshold`?